src/components/task-teacher-review/crumb-mixin.cjsx

```diff
@@ -66,16 +66,29 @@ module.exports =
       _.each crumb.data, (data) ->
         data.key = crumb.key
     else
-      crumb.data.key = crumb.key
+      crumb.data.key = index
 
 
   _getCrumbsForHomework: (stats) ->
     exercises = @_getExercisesFromStats(stats)
     crumbs = _.chain(exercises)
-      .map(@_makeCrumb)
-      .each(@_indexCrumb)
+      .map((data) =>
+        {questions} = JSON.parse(data.content)
+        stepCrumbs = []
```

Comment:
Hmm.  Do you really need `JSON.parse` ?  I stuck a console.log in here and it looks to me like `data.content` is already an object.  That was when loading the metrics for an assignment as a teacher.